apps/public-docsite-v9/src/Concepts/Slots/Slots.stories.mdx

```diff
@@ -0,0 +1,379 @@
+  For example, if you want to have borders to be a specific color on `:hover`, you can create a theme that sets `colorNeutralStroke1Hover`.
+
+- If you need to slightly adjust the display of a specific component instance, prefer to apply a custom style.
+  For example, if you want to make a particular component have a thicker border.
+  You create the style using `makeStyles` and/or `mergeClasses` and then apply it to the component using `className`.
+
+- If you want to change how a component behaves, make deep style customizations,
+  replace non-slot parts, or wrap a component with different props, then consider using the hooks API.
+  The hooks API gives you complete control to recompose a component but is more complex than using slots.
+
+### Conditional rendering
+
+Some components conditionally render slots.
+
+For example, Avatar has `label` slot that only renders when there is no image provided.
+It also has an `icon` slot that only renders when there is neither an image nor a name provided.
+
+### What about children?
+
+The primary content within a component is defined by adding children.
+If you like, you can think of children as an unnamed slot.
+
+Component children allow for building deeper hierarchies using JSX/TSX.
+They also allow for greater extension of the types of content.
+
+For example, Menu is a hierarchy of menu items.
+Each level of submenu can contain different trigger configurations, and
+the type of menu items in the list can vary.
+
+```tsx
+<Menu>
+  <MenuTrigger>
+    <Button>Toggle menu</Button>
+  </MenuTrigger>
+  <MenuPopover>
+    <MenuList>
+      <MenuGroup>
+        <MenuGroupHeader>Create New Items</MenuGroupHeader>
+        <MenuItem>New Project</MenuItem>
+        <MenuItem>New Timeline</MenuItem>
+      </MenuGroup>
+      <MenuDivider />
+      <MenuItem disabled>Open File...</MenuItem>
+      <MenuItem>Open Folder...</MenuItem>
+      <MenuDivider />
+      <Menu>
+        <MenuTrigger>
+          <MenuItem>Zoom</MenuItem>
+        </MenuTrigger>
+        <MenuPopover>
+          <MenuList>
+            <MenuItem>Zoom In</MenuItem>
+            <MenuItem>Zoom Out</MenuItem>
+          </MenuList>
+        </MenuPopover>
+      </Menu>
+    </MenuList>
+  </MenuPopover>
+</Menu>
+```
+
+## Usage examples
+
+### Passing content
+
+You can pass a primitive value to slots.
+The `Input` component's `contentBefore` and `contentAfter` slots can be passed strings.
+
+```tsx
+<Input contentBefore="$" value="10" contentAfter=".00" />
+```
+
+You can pass HTML values to slots.
+The `Button` component can be passed an `img`.
+
+```tsx
+<Button icon={<img src='site-icon.png' alt='branded site icon' /> }
+```
+
+You can pass JSX/TSX to slots.
+The `Button` component can be passed an `CalendarRegular24` icon.
+
+```tsx
+<Button icon={<CalendarRegular24 />}
+```
+
+### Passing props
+
+When a slot renders a component or element by default, you can pass props to it.
```

Comment:
- `Component` - is a React component
- `<Component />` - is a JSX element

If a React component will be passed to a slot i.e. `<Button icon={Component} />` it will explode 💣 

Let's be more accurate with terms 🐱 

```suggestion
When a slot renders an element by default, you can pass props to it.
```